alarms/alarms.py

```diff
@@ -60,6 +60,27 @@
 MARKER_SIMPLIFICATION = re.compile(r'{(\w*):?.*?}')
 
 
+def read_settings():
+    settings = {}
+    expected_elements = (
+        'smtp_host', 'db_host', 'log_file_name', 'db_name',
+        'reader_user', 'reader_pw', 'alarm_user', 'alarm_pw',
+        'error_contact_email', 'mail_sender'
+    )
+    system_global = xml.etree.ElementTree.ElementTree()
+    conf = pathlib.Path(__file__).resolve().parent.parent / 'global_settings.xml'
+    system_global.parse(conf)
+    system_global = system_global.getroot()
+
```

Comment:
This may be unimportant. But if there is no `mail_alarm_settings`, you may end returning weird results if whatever the last child of global was happens to share a item name with your alarm keys and if there is nothing in globals settings at all you will get an exception. Both concerns could be fixed by attaching an early return if no `mail_alarm_settings` item is found:
```python
    for child in system_global:
        if child.tag == 'mail_alarm_settings':
            break
    else:
        return settings
```

Reply:
I am not too concerned, we provide a template, that really should be used. However, I have added your suggestion, if for nothing else, to try to use the for-else syntax (for the first time I believe:) )

Reply:
If you squint, that "else" almost looks like the non-existent keyword "nobreak" ;)